.gitignore

```diff
@@ -29,3 +29,5 @@ xb526qv4524_05_0001.jp2
 /python*
 
 todo.md
+
+.hypothesis
```

Comment:
This is a directory uses for a bit of bookkeeping and maintenance, but its contents aren’t generally human-readable or something that needs to be checked in.